docs/migrate-istio-to-asm/traffic-split/virtual-service.yaml

```diff
@@ -0,0 +1,23 @@
+# This VirtualService routes the example.com traffic from my-gateway
+# to port 80 of the hello-world-service.
+apiVersion: networking.istio.io/v1alpha3
```

Comment:
region tag would be: `service_mesh_traffic_split_virtualservice_frontend_ingress`

Reply:
Done. ✅  Thanks for formulating the tag for me.
Heads-up: Instead of
`service_mesh_traffic_split_virtualservice_frontend_ingress`, I used
`service_mesh_traffic_split_virtual_service_virtualservice_frontend_ingress` to include the file name.